### DISCUSSION TITLE
An question about cargo run --release on win10

### DISCUSSION THREAD

**Discussion by jasper2085** [rustdesk/rustdesk#747] [Q&A]
I built it with cargo run --release on win10. The result is that it can be started normally when running, but the target/release/rustdesk.exe will report an error when it is copied to another computer, saying that not found src/ui/index html. 
<img width="856" alt="图片" src="https://user-images.githubusercontent.com/106738381/173219797-0ad359af-0704-4f4f-bfbf-5a18c3287f73.png">

**Comment by tahssinaldrajee** [comment]
same problem

**Comment by 21pages** [comment]
run `build.py`

**Reply by tahssinaldrajee** [reply]
with python build.py this errors appear : -
error[E0433]: failed to resolve: use of undeclared type `FileType`
  --> libs\hbb_common\src\fs.rs:32:33
   |
32 |                     entry_type: FileType::DirDrive.into(),
   |                                 ^^^^^^^^ use of undeclared type `FileType`

error[E0433]: failed to resolve: use of undeclared type `FileType`
  --> libs\hbb_common\src\fs.rs:72:26
   |
72 |                         (FileType::DirLink.into(), 0)
   |                          ^^^^^^^^ use of undeclared type `FileType`

error[E0433]: failed to resolve: use of undeclared type `FileType`
  --> libs\hbb_common\src\fs.rs:74:26
   |
74 |                         (FileType::Dir.into(), 0)
   |                          ^^^^^^^^ use of undeclared type `FileType`

error[E0433]: failed to resolve: use of undeclared type `FileType`
  --> libs\hbb_common\src\fs.rs:78:26
   |
78 |                         (FileType::FileLink.into(), 0)
   |                          ^^^^^^^^ use of undeclared type `FileType`

error[E0433]: failed to resolve: use of undeclared type `FileType`
  --> libs\hbb_common\src\fs.rs:80:26
   |
80 |                         (FileType::File.into(), meta.len())
   |                          ^^^^^^^^ use of undeclared type `FileType`

error[E0433]: failed to resolve: use of undeclared type `FileType`
   --> libs\hbb_common\src\fs.rs:140:20
    |
140 |                 Ok(FileType::File) => {
    |                    ^^^^^^^^ use of undeclared type `FileType`

error[E0433]: failed to resolve: use of undeclared type `FileType`
   --> libs\hbb_common\src\fs.rs:145:20
    |
145 |                 Ok(FileType::Dir) => {
    |                    ^^^^^^^^ use of undeclared type `FileType`

error[E0433]: failed to resolve: use of undeclared type `FileType`
   --> libs\hbb_common\src\fs.rs:176:25
    |
176 |             entry_type: FileType::File.into(),
    |                         ^^^^^^^^ use of undeclared type `FileType`

error[E0433]: failed to resolve: use of undeclared type `FileResponse`
   --> libs\hbb_common\src\fs.rs:509:24
    |
509 |         let mut resp = FileResponse::new();
    |                        ^^^^^^^^^^^^ use of undeclared type `FileResponse`

error[E0433]: failed to resolve: use of undeclared crate or module `file_transfer_send_confirm_request`
   --> libs\hbb_common\src\fs.rs:571:22
    |
571 |                 Some(file_transfer_send_confirm_request::Union::Skip(s)) => {
    |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `file_transfer_send_confirm_request`

error[E0433]: failed to resolve: use of undeclared crate or module `file_transfer_send_confirm_request`
   --> libs\hbb_common\src\fs.rs:579:22
    |
579 |                 Some(file_transfer_send_confirm_request::Union::OffsetBlk(_offset)) => {
    |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `file_transfer_send_confirm_request`

error[E0433]: failed to resolve: use of undeclared type `FileResponse`
   --> libs\hbb_common\src\fs.rs:603:20
    |
603 |     let mut resp = FileResponse::new();
    |                    ^^^^^^^^^^^^ use of undeclared type `FileResponse`

error[E0433]: failed to resolve: use of undeclared type `FileResponse`
   --> libs\hbb_common\src\fs.rs:617:20
    |
617 |     let mut resp = FileResponse::new();
    |                    ^^^^^^^^^^^^ use of undeclared type `FileResponse`

error[E0433]: failed to resolve: use of undeclared type `FileResponse`
   --> libs\hbb_common\src\fs.rs:631:20
    |
631 |     let mut resp = FileResponse::new();
    |                    ^^^^^^^^^^^^ use of undeclared type `FileResponse`

error[E0433]: failed to resolve: use of undeclared type `FileAction`
   --> libs\hbb_common\src\fs.rs:641:22
    |
641 |     let mut action = FileAction::new();
    |                      ^^^^^^^^^^ use of undeclared type `FileAction`

error[E0433]: failed to resolve: use of undeclared type `FileAction`
   --> libs\hbb_common\src\fs.rs:649:22
    |
649 |     let mut action = FileAction::new();
    |                      ^^^^^^^^^^ use of undeclared type `FileAction`

error[E0433]: failed to resolve: use of undeclared type `FileAction`
   --> libs\hbb_common\src\fs.rs:665:22
    |
665 |     let mut action = FileAction::new();
    |                      ^^^^^^^^^^ use of undeclared type `FileAction`

error[E0433]: failed to resolve: use of undeclared type `FileResponse`
   --> libs\hbb_common\src\fs.rs:680:20
    |
680 |     let mut resp = FileResponse::new();
    |                    ^^^^^^^^^^^^ use of undeclared type `FileResponse`

error[E0433]: failed to resolve: use of undeclared type `FileType`
   --> libs\hbb_common\src\fs.rs:740:16
    |
740 |             Ok(FileType::Dir) => {
    |                ^^^^^^^^ use of undeclared type `FileType`

error[E0433]: failed to resolve: use of undeclared type `FileType`
   --> libs\hbb_common\src\fs.rs:743:16
    |
743 |             Ok(FileType::DirLink) | Ok(FileType::FileLink) => {
    |                ^^^^^^^^ use of undeclared type `FileType`

error[E0433]: failed to resolve: use of undeclared type `FileType`
   --> libs\hbb_common\src\fs.rs:743:40
    |
743 |             Ok(FileType::DirLink) | Ok(FileType::FileLink) => {
    |                                        ^^^^^^^^ use of undeclared type `FileType`

error[E0412]: cannot find type `FileDirectory` in this scope
  --> libs\hbb_common\src\fs.rs:16:69
   |
16 | pub fn read_dir(path: &PathBuf, include_hidden: bool) -> ResultType<FileDirectory> {
   |                -                                                    ^^^^^^^^^^^^^ not found in this scope
   |                |
   |                help: you might be missing a type parameter: `<FileDirectory>`

error[E0422]: cannot find struct, variant or union type `FileDirectory` in this scope
  --> libs\hbb_common\src\fs.rs:17:19
   |
17 |     let mut dir = FileDirectory {
   |                   ^^^^^^^^^^^^^ not found in this scope

error[E0422]: cannot find struct, variant or union type `FileEntry` in this scope
  --> libs\hbb_common\src\fs.rs:30:34
   |
30 |                 dir.entries.push(FileEntry {
   |                                  ^^^^^^^^^ not found in this scope

error[E0422]: cannot find struct, variant or union type `FileEntry` in this scope
  --> libs\hbb_common\src\fs.rs:92:30
   |
92 |             dir.entries.push(FileEntry {
   |                              ^^^^^^^^^ not found in this scope

error[E0412]: cannot find type `FileEntry` in this scope
   --> libs\hbb_common\src\fs.rs:132:21
    |
132 | ) -> ResultType<Vec<FileEntry>> {
    |                     ^^^^^^^^^
    |
help: a trait with a similar name exists
    |
132 | ) -> ResultType<Vec<FileExt>> {
    |                     ~~~~~~~
help: you might be missing a type parameter
    |
128 | fn read_dir_recursive<FileEntry>(
    |                      +++++++++++

error[E0422]: cannot find struct, variant or union type `FileEntry` in this scope
   --> libs\hbb_common\src\fs.rs:175:20
    |
175 |         files.push(FileEntry {
    |                    ^^^^^^^^^ not found in this scope

error[E0412]: cannot find type `FileEntry` in this scope
   --> libs\hbb_common\src\fs.rs:187:80
    |
187 | pub fn get_recursive_files(path: &str, include_hidden: bool) -> ResultType<Vec<FileEntry>> {
    |                                                                                ^^^^^^^^^
    |
help: a trait with a similar name exists
    |
187 | pub fn get_recursive_files(path: &str, include_hidden: bool) -> ResultType<Vec<FileExt>> {
    |                                                                                ~~~~~~~
help: you might be missing a type parameter
    |
187 | pub fn get_recursive_files<FileEntry>(path: &str, include_hidden: bool) -> ResultType<Vec<FileEntry>> {
    |                           +++++++++++

error[E0412]: cannot find type `FileEntry` in this scope
   --> libs\hbb_common\src\fs.rs:210:20
    |
210 |     pub files: Vec<FileEntry>,
    |                    ^^^^^^^^^
    |
help: a trait with a similar name exists
    |
210 |     pub files: Vec<FileExt>,
    |                    ~~~~~~~
help: you might be missing a type parameter
    |
202 | pub struct TransferJob<FileEntry> {
    |                       +++++++++++

error[E0412]: cannot find type `FileEntry` in this scope
   --> libs\hbb_common\src\fs.rs:278:20
    |
278 |         files: Vec<FileEntry>,
    |                    ^^^^^^^^^
    |
help: a trait with a similar name exists
    |
278 |         files: Vec<FileExt>,
    |                    ~~~~~~~
help: you might be missing a type parameter
    |
270 | impl<FileEntry> TransferJob {
    |     +++++++++++

error[E0412]: cannot find type `FileEntry` in this scope
   --> libs\hbb_common\src\fs.rs:324:33
    |
324 |     pub fn files(&self) -> &Vec<FileEntry> {
    |                                 ^^^^^^^^^
    |
help: a trait with a similar name exists
    |
324 |     pub fn files(&self) -> &Vec<FileExt> {
    |                                 ~~~~~~~
help: you might be missing a type parameter
    |
270 | impl<FileEntry> TransferJob {
    |     +++++++++++

error[E0412]: cannot find type `FileEntry` in this scope
   --> libs\hbb_common\src\fs.rs:329:44
    |
329 |     pub fn set_files(&mut self, files: Vec<FileEntry>) {
    |                                            ^^^^^^^^^
    |
help: a trait with a similar name exists
    |
329 |     pub fn set_files(&mut self, files: Vec<FileExt>) {
    |                                            ~~~~~~~
help: you might be missing a type parameter
    |
270 | impl<FileEntry> TransferJob {
    |     +++++++++++

error[E0412]: cannot find type `FileTransferBlock` in this scope
   --> libs\hbb_common\src\fs.rs:383:42
    |
383 |     pub async fn write(&mut self, block: FileTransferBlock) -> ResultType<()> {
    |                                          ^^^^^^^^^^^^^^^^^ not found in this scope

error[E0412]: cannot find type `FileTransferBlock` in this scope
   --> libs\hbb_common\src\fs.rs:426:76
    |
270 | impl TransferJob {
    |     - help: you might be missing a type parameter: `<FileTransferBlock>`
...
426 |     pub async fn read(&mut self, stream: &mut Stream) -> ResultType<Option<FileTransferBlock>> {
    |                                                                            ^^^^^^^^^^^^^^^^^ not found in this scope

error[E0422]: cannot find struct, variant or union type `FileTransferBlock` in this scope
   --> libs\hbb_common\src\fs.rs:498:17
    |
498 |         Ok(Some(FileTransferBlock {
    |                 ^^^^^^^^^^^^^^^^^ not found in this scope

error[E0433]: failed to resolve: use of undeclared type `Message`
   --> libs\hbb_common\src\fs.rs:508:23
    |
508 |         let mut msg = Message::new();
    |                       ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0422]: cannot find struct, variant or union type `FileTransferDigest` in this scope
   --> libs\hbb_common\src\fs.rs:515:25
    |
515 |         resp.set_digest(FileTransferDigest {
    |                         ^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0412]: cannot find type `FileTransferSendConfirmRequest` in this scope
   --> libs\hbb_common\src\fs.rs:566:35
    |
566 |     pub fn confirm(&mut self, r: &FileTransferSendConfirmRequest) -> bool {
    |                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0412]: cannot find type `Message` in this scope
   --> libs\hbb_common\src\fs.rs:602:79
    |
602 | pub fn new_error<T: std::string::ToString>(id: i32, err: T, file_num: i32) -> Message {
    |                                                                               ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0422]: cannot find struct, variant or union type `FileTransferError` in this scope
   --> libs\hbb_common\src\fs.rs:604:20
    |
604 |     resp.set_error(FileTransferError {
    |                    ^^^^^^^^^^^^^^^^^ not found in this scope

error[E0433]: failed to resolve: use of undeclared type `Message`
   --> libs\hbb_common\src\fs.rs:610:23
    |
610 |     let mut msg_out = Message::new();
    |                       ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0412]: cannot find type `FileEntry` in this scope
   --> libs\hbb_common\src\fs.rs:616:50
    |
616 | pub fn new_dir(id: i32, path: String, files: Vec<FileEntry>) -> Message {
    |                                                  ^^^^^^^^^
    |
help: a trait with a similar name exists
    |
616 | pub fn new_dir(id: i32, path: String, files: Vec<FileExt>) -> Message {
    |                                                  ~~~~~~~
help: you might be missing a type parameter
    |
616 | pub fn new_dir<FileEntry>(id: i32, path: String, files: Vec<FileEntry>) -> Message {
    |               +++++++++++

error[E0412]: cannot find type `Message` in this scope
   --> libs\hbb_common\src\fs.rs:616:65
    |
616 | pub fn new_dir(id: i32, path: String, files: Vec<FileEntry>) -> Message {
    |                                                                 ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0422]: cannot find struct, variant or union type `FileDirectory` in this scope
   --> libs\hbb_common\src\fs.rs:618:18
    |
618 |     resp.set_dir(FileDirectory {
    |                  ^^^^^^^^^^^^^ not found in this scope

error[E0433]: failed to resolve: use of undeclared type `Message`
   --> libs\hbb_common\src\fs.rs:624:23
    |
624 |     let mut msg_out = Message::new();
    |                       ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0412]: cannot find type `FileTransferBlock` in this scope
   --> libs\hbb_common\src\fs.rs:630:25
    |
630 | pub fn new_block(block: FileTransferBlock) -> Message {
    |                         ^^^^^^^^^^^^^^^^^ not found in this scope

error[E0412]: cannot find type `Message` in this scope
   --> libs\hbb_common\src\fs.rs:630:47
    |
630 | pub fn new_block(block: FileTransferBlock) -> Message {
    |                                               ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0433]: failed to resolve: use of undeclared type `Message`
   --> libs\hbb_common\src\fs.rs:633:23
    |
633 |     let mut msg_out = Message::new();
    |                       ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0412]: cannot find type `FileTransferSendConfirmRequest` in this scope
   --> libs\hbb_common\src\fs.rs:639:28
    |
639 | pub fn new_send_confirm(r: FileTransferSendConfirmRequest) -> Message {
    |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0412]: cannot find type `Message` in this scope
   --> libs\hbb_common\src\fs.rs:639:63
    |
639 | pub fn new_send_confirm(r: FileTransferSendConfirmRequest) -> Message {
    |                                                               ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0433]: failed to resolve: use of undeclared type `Message`
   --> libs\hbb_common\src\fs.rs:640:23
    |
640 |     let mut msg_out = Message::new();
    |                       ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0412]: cannot find type `FileEntry` in this scope
   --> libs\hbb_common\src\fs.rs:648:69
    |
648 | pub fn new_receive(id: i32, path: String, file_num: i32, files: Vec<FileEntry>) -> Message {
    |                                                                     ^^^^^^^^^
    |
help: a trait with a similar name exists
    |
648 | pub fn new_receive(id: i32, path: String, file_num: i32, files: Vec<FileExt>) -> Message {
    |                                                                     ~~~~~~~
help: you might be missing a type parameter
    |
648 | pub fn new_receive<FileEntry>(id: i32, path: String, file_num: i32, files: Vec<FileEntry>) -> Message {
    |                   +++++++++++

error[E0412]: cannot find type `Message` in this scope
   --> libs\hbb_common\src\fs.rs:648:84
    |
648 | pub fn new_receive(id: i32, path: String, file_num: i32, files: Vec<FileEntry>) -> Message {
    |                                                                                    ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0422]: cannot find struct, variant or union type `FileTransferReceiveRequest` in this scope
   --> libs\hbb_common\src\fs.rs:650:24
    |
650 |     action.set_receive(FileTransferReceiveRequest {
    |                        ^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0433]: failed to resolve: use of undeclared type `Message`
   --> libs\hbb_common\src\fs.rs:657:23
    |
657 |     let mut msg_out = Message::new();
    |                       ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0412]: cannot find type `Message` in this scope
   --> libs\hbb_common\src\fs.rs:663:80
    |
663 | pub fn new_send(id: i32, path: String, file_num: i32, include_hidden: bool) -> Message {
    |                                                                                ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0422]: cannot find struct, variant or union type `FileTransferSendRequest` in this scope
   --> libs\hbb_common\src\fs.rs:666:21
    |
666 |     action.set_send(FileTransferSendRequest {
    |                     ^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0433]: failed to resolve: use of undeclared type `Message`
   --> libs\hbb_common\src\fs.rs:673:23
    |
673 |     let mut msg_out = Message::new();
    |                       ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0412]: cannot find type `Message` in this scope
   --> libs\hbb_common\src\fs.rs:679:44
    |
679 | pub fn new_done(id: i32, file_num: i32) -> Message {
    |                                            ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0422]: cannot find struct, variant or union type `FileTransferDone` in this scope
   --> libs\hbb_common\src\fs.rs:681:19
    |
681 |     resp.set_done(FileTransferDone {
    |                   ^^^^^^^^^^^^^^^^ not found in this scope

error[E0433]: failed to resolve: use of undeclared type `Message`
   --> libs\hbb_common\src\fs.rs:686:23
    |
686 |     let mut msg_out = Message::new();
    |                       ^^^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
2   | use crate::protobuf::Message;
    |
2   | use protobuf::Message;
    |

error[E0412]: cannot find type `FileEntry` in this scope
   --> libs\hbb_common\src\fs.rs:766:49
    |
766 | pub fn transform_windows_path(entries: &mut Vec<FileEntry>) {
    |                                                 ^^^^^^^^^
    |
help: a trait with a similar name exists
    |
766 | pub fn transform_windows_path(entries: &mut Vec<FileExt>) {
    |                                                 ~~~~~~~
help: you might be missing a type parameter
    |
766 | pub fn transform_windows_path<FileEntry>(entries: &mut Vec<FileEntry>) {
    |                              +++++++++++

error[E0412]: cannot find type `FileTransferDigest` in this scope
   --> libs\hbb_common\src\fs.rs:774:17
    |
774 |     NeedConfirm(FileTransferDigest),
    |                 ^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0412]: cannot find type `FileTransferDigest` in this scope
   --> libs\hbb_common\src\fs.rs:781:14
    |
781 |     digest: &FileTransferDigest,
    |              ^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0422]: cannot find struct, variant or union type `FileTransferDigest` in this scope
   --> libs\hbb_common\src\fs.rs:792:43
    |
792 |         Ok(DigestCheckResult::NeedConfirm(FileTransferDigest {
    |                                           ^^^^^^^^^^^^^^^^^^ not found in this scope

warning: unused import: `message_proto::*`
 --> libs\hbb_common\src\fs.rs:9:39
  |
9 | use crate::{bail, get_version_number, message_proto::*, ResultType, Stream};
  |                                       ^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

error[E0308]: mismatched types
   --> libs\hbb_common\src\fs.rs:489:36
    |
489 |             if !is_compressed_file(name) {
    |                 ------------------ ^^^^
    |                 |                  |
    |                 |                  expected `&str`, found `str`
    |                 |                  help: consider borrowing here: `&name`
    |                 arguments to this function are incorrect
    |
note: function defined here
   --> libs\hbb_common\src\fs.rs:257:4
    |
257 | fn is_compressed_file(name: &str) -> bool {
    |    ^^^^^^^^^^^^^^^^^^ ----------

error[E0283]: type annotations needed
   --> libs\hbb_common\src\fs.rs:210:5
    |
201 | #[derive(Default)]
    |          ------- in this derive macro expansion
...
210 |     pub files: Vec<FileEntry>,
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^ cannot infer type
    |
    = note: cannot satisfy `_: Default`
    = note: this error originates in the derive macro `Default` (in Nightly builds, run with -Z macro-backtrace for more info)

Some errors have detailed explanations: E0283, E0308, E0412, E0422, E0433.
For more information about an error, try `rustc --explain E0283`.
warning: `hbb_common` (lib) generated 1 warning
error: could not compile `hbb_common` due to 67 previous errors; 1 warning emitted
warning: build failed, waiting for other jobs to finish...
mv: cannot stat 'target/release/rustdesk.exe': No such file or directory
cp: cannot stat 'target/release/RustDesk.exe': No such file or directory
Build with features ['PrivacyMode']
PrivacyMode download begin
PrivacyMode download end. extract bein
PrivacyMode extract end
features: ['inline', 'with_rc']
Not signed

**Comment by ion-tichy** [comment]
same problem when using the `rustdesk-builder` docker container under Linux (Manjaro):

```
WARNING:DOM: failed to load "<path_to_rustdesk_repo>/target/release/src/ui/index.html" file, error=2
```

**Comment by ion-tichy** [comment]
managed to "fix" this by simply copying the contents of `src/ui` into `target/release/src/ui`.

Is this the intended way of building a new client release? because this is far from a "standalone executable".

**Comment by yashpatel1598** [comment]
hi how to generate release in linux machine 
so we can share that executable directly and they dont have to build anything and directly use that executable file.
